src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportOrchestratorJobResult.cs

```diff
@@ -54,5 +54,10 @@ public class ImportOrchestratorJobResult
         public int CreatedJobCount { get; set; } // TODO: remove in stage 3
 
         public long? TotalSizeInBytes { get; set; } // TODO: remove in stage 3
+
+        /// <summary>
+        /// Import mode.
+        /// </summary>
+        public ImportMode ImportMode { get; set; }
```

Comment:
Is import mode available in the job definition?

Reply:
@SergeyGaluzo  removed